acceptance/openstack/keymanager/v1/quotas_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build acceptance || keymanager || orders
+// +build acceptance keymanager orders
+
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/projects"
+	"github.com/gophercloud/gophercloud/openstack/keymanager/v1/quotas"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestQuotasCRUD(t *testing.T) {
+	clients.RequireAdmin(t)
```

Comment:
weird, this test is failing despite you're an admin:

=== RUN   TestQuotasCRUD
    convenience.go:35: Failure in quotas_test.go, line 46: unexpected error "Request forbidden: [PUT http://10.1.0.42/key-manager/v1/project-quotas/052045906ba04ff596538b79f9bc8288], error message: {\"code\": 403, \"title\": \"Forbidden\", \"description\": \"Project Quotas attempt not allowed - please review your user/project privileges\"}"